Formula/step.rb

```diff
@@ -0,0 +1,45 @@
+class Step < Formula
+  desc "Crypto and x509 Swiss-Army-Knife"
+  homepage "https://smallstep.com"
+  url "https://github.com/smallstep/cli/releases/download/v0.8.3/brew_step_0.8.3.tar.gz"
```

Comment:
What does this provide that `https://github.com/smallstep/cli/archive/v0.8.4-rc.4.tar.gz` does not?

Reply:
@MikeMcQuaid It appears to me that it's just a tarball containing https://github.com/smallstep/cli AND https://github.com/smallstep/certificates and then the formula tells go to build and install both.

Reply:
The release ships only the compiled binary, and only for cli.

Reply:
I think it would be preferable to set the URL to a repo tarball and add the certificates as a `resource` rather than have a special Homebrew tarball.

Reply:
Yeah, I agree, that would make it clearer to maintainers what exactly is going on. The inception double-repo tarball is a bit opaque/mystifying.

Reply:
Yep, using a resource made way more sense and also simplified our build pipeline.